.settings/org.eclipse.jdt.apt.core.prefs

```diff
@@ -0,0 +1,2 @@
+eclipse.preferences.version=1
```

Comment:
Please add the `.settings` folder to `.gitignore`. All mentioned additions for `.gitIgnore` are your specific eclipse configuration of the project and should not be committed ;)

Reply:
hey , i am sorry i am new but , i din't use eclipse , where can i get eclipse config files ?

Reply:
hey , i am sorry i am new but , i din't use eclipse , where can i get eclipse config files ?

Reply:
no problem :) I will help you and write a more detailed answer